packages/grid/demo/stories/GridStory.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import {
+  GridContainer,
+  IGridContainerProps,
+  IUseGridProps,
+  IUseGridReturnValue,
+  useGrid
+} from '@zendeskgarden/container-grid';
+
+interface IComponent extends IUseGridProps, IUseGridReturnValue {}
+
+const Component = ({ rtl, matrix, selection, getGridCellProps }: IComponent) => (
+  <table role="grid" style={{ direction: rtl ? 'rtl' : 'ltr' }}>
+    <tbody>
+      {matrix.map((row, rowIdx) => (
+        <tr key={rowIdx}>
+          {row.map((column, colIdx) => {
+            const {
+              role,
+              'aria-selected': selected, // the gridcell itself is not selectable in this implementation
+              ...props
+            } = getGridCellProps({ rowIdx, colIdx });
```

Comment:
This prop extraction and subsequent usage of the current grid's `selection` concept below is a key demo differentiation that corrects previous invalid ARIA and points the way for how we'll address an updated `ColorSwatch` in `react-components` by refactoring to use radio buttons (or checkboxes, depending on design) for selection.